client/src/components/Home/style.css

```diff
@@ -0,0 +1,33 @@
+.home {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 70%;
```

Comment:
I think not so good use 70% of height their logo and hamburger-button and footer and may some problem when open sidebar 
just work without height.